pkg/client/listers/eventing/v1alpha1/clusterprovisioner.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by lister-gen. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	v1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/tools/cache"
+)
+
+// ClusterProvisionerLister helps list ClusterProvisioners.
+type ClusterProvisionerLister interface {
+	// List lists all ClusterProvisioners in the indexer.
+	List(selector labels.Selector) (ret []*v1alpha1.ClusterProvisioner, err error)
+	// Get retrieves the ClusterProvisioner from the index for a given name.
+	Get(name string) (*v1alpha1.ClusterProvisioner, error)
```

Comment:
It looks like this is autogenned correctly, if you compare with https://github.com/knative/eventing/blob/6a6778ac808dbda0798420d7fe1cde0ac80de9ed/pkg/client/listers/eventing/v1alpha1/subscription.go#L33

Reply:
Seems so! I guess list objects inherit `nonNamespaced` from their element types.